docs/docs/language_reference/schema/index.md

```diff
@@ -116,19 +116,272 @@ type Post implements Webpage {
 
 ## Vertex types and properties
 
+A [vertex][vertex] is akin to a table in SQL, given our Flutter example where
+we have users and they may post things, or follow other users we can see we have
+two clear vertices. A User vertex and a Post vertex.
+
+A vertex can have properties associated with it (similar to columns in SQL), and
+edges. We'll initially just focus on properties as these are the simplest case.
+For a user we can imagine the following properties as a minimum:
+
+* `url`: the URL to the users profile page (REQUIRED)
+* `username`: the unique username of the account (REQUIRED)
+* `display_name`: the name we display for the user (OPTIONAL)
+
+With these two properties we can define a type for the schema called `User` which
+looks as follows:
+
+```graphql
+"""
+A user account on Flutter.
+"""
+type User {
+    """
+    The URL of this user's profile page.
+    """
+    url: String!
+
+    """
+    The user's unique username. All users are required to have one.
+    """
+    username: String!
+
+    """
+    The name the user would like to be known by, if any was set.
+
+    If the user has not set a display name, this value will be `null`.
+    """
+    display_name: String
+}
+```
+
+Here we've defined three string properties with doc comments to describe them. We've 
+also made `url` and `username` required fields by adding the `!` suffix to the type. This
+means that the value can't be null. Alternatively, `display_name` is allowed to be
+`null` as it is just a `String`.
+
+We'll now also define our `Post` type, this will be very similar with a url to the post and
+a message representing the post. For now the user who posted it will be omitted and covered
+in the Edges section.
```

Comment:
```suggestion
We'll now also define our `Post` type: it will be very similar, with a url to the post and
a message representing the post. For now, the user who posted it will be omitted and covered
in the next section.
```